src/data/data.js

```diff
@@ -22,56 +32,59 @@ const pageData = {
       {
         name: 'headerTitleComp',
         title: 'Iniciativa',
-        header: 'Iniciativa'
+        header: 'Iniciativa',
       },
       {
         name: 'textBlockComp',
         backgroundColor: 'white',
-        title: '¿Sabías que 1 litro de aceite contamina más de 1000 litros de agua?',
+        title:
+          '¿Sabías que 1 litro de aceite contamina más de 1000 litros de agua?',
         text:
-          'Hemos ideado una manera de deshacernos del aceite de manera responsable con el medio ambiente.'
+          'Hemos ideado una manera de deshacernos del aceite de manera responsable con el medio ambiente.',
       },
       {
         name: 'textBlockComp',
         backgroundColor: 'green',
-        title: '¿Sabías que 1 litro de aceite contamina más de 1000 litros de agua?'
+        title:
+          '¿Sabías que 1 litro de aceite contamina más de 1000 litros de agua?',
       },
       {
         name: 'timelineHeader',
-        title: 'Como recoger el aceite'
+        title: 'Como recoger el aceite',
       },
       {
         name: 'timelineItem',
         index: 0,
         title: 'Deja enfriar el aceite',
         text: 'Con esto evitamos posibles quemaduras o derretir el embase.',
-        icon: ''
+        icon: ',
```

Comment:
Don't remove the closing quote on this property, otherwise it will cause an error. Leave the icon value as it was before

Reply:
resolved